jsx/StaticDataTable.js

```diff
@@ -295,6 +295,68 @@ class StaticDataTable extends Component {
     });
   }
 
+  /**
+   * Get if the current sorting column has mixed types in its values.
+   *
+   * @return {bool} true if mixed types, else false.
+   */
+  hasMixedTypes() {
+    // TODO: data column type check should probably be done once at init,
+    // meaning when receiving data, to find which columns have mixed types.
+    let typesFound = null;
+
+    // not the default column
+    if (this.state.SortColumn === -1) {
+      return false;
+    }
```

Comment:
To ensure even better isolation, I would personally move this check out of the function, and make the function take the column index as a parameter. The result would look like something like this :

```ts
let isMixedType = this.state.SortColumn !== -1 && this.hasMixedTypes(ths.state.SortColumn);
```

Reply:
I get the utils function target here. I am pretty new to React, for what I can understand, state and props are accessible throughout a component. If the function has to be isolated, shouldn't it also receive props.Data as parameter?
For now, I do not want to disconnect this section too much from the original loop, I will just let it as is.

Reply:
By isolated, I don't mean outside the component (it is still a method in this design), but rather loosely coupled as would not depend only depend on the data and not on the SortColumn anymore. Other than that, you're right on the state and props.

Anyway, the code looks ok to me. I think if we want something really clean we would need to refactor the full `StaticDataTable` but this is a job for another day.